### DISCUSSION TITLE
Disabling "series hover label"

### DISCUSSION THREAD

**Discussion by pavlexander** [Live-Charts/LiveCharts2#839] [Q&A] [answered]
I was just  looking into possibility to disable the "series hover label".. Sorry I have no idea how to describe it better. Here is the picture.

![image](https://user-images.githubusercontent.com/20644772/207060896-e199e372-e1f3-4d4b-8cf3-7cbec6072e90.png)

Here is the source code

```csharp
            var candles = GetCanles();
            var plottableCandles = LCConverter.ConvertToPlottalble(candles);

            var form = new LCChartForm();
            var plot = form.MainPlot;
            form.Show();

            LiveChartsCore.SkiaSharpView.Axis[] axis = {
                new LiveChartsCore.SkiaSharpView.Axis
                {
                    LabelsRotation = 45,
                    Labeler = value => new DateTime((long)value).ToString("yyyy MMM dd HH mm"),
                    UnitWidth = TimeSpan.FromMinutes(1).Ticks,
                }
            };

            plot.XAxes = axis;
            plot.Series = new ISeries[]
            {
                plottableCandles
            };
```

the code repo is here just in case: https://github.com/pavlexander/plot_test

**Comment by mjanulaitis** [comment]
I'd like to change the size of the font.  I'm not sure how to do that.  Any idea's?

**Reply by mjanulaitis** [reply]
TooltipTextSize

**Comment by devinvisible** [comment, marked as answer]
Is `plot.TooltipPosition = LiveChartsCore.Measure.TooltipPosition.Hidden` what you're looking for?

**Comment by babylard** [comment]
Not exactly a good idea to fuck with Hackers.

Sent with [Proton Mail](https://proton.me/) secure email.

------- Original Message -------
